webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBarActions.test.js.snap

```diff
@@ -61,7 +67,7 @@ Array [
           },
         ],
         "page": 1,
-        "pages": 1.5,
+        "pages": NaN,
```

Comment:
is this a bug?

Reply:
yes, I didn't update the fixtures with `subtotal` prop instead of `total`.